src/main/java/dataloader/CsvDataLoader.java

```diff
@@ -0,0 +1,71 @@
+package dataloader;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import tech.tablesaw.api.Table;
+import tech.tablesaw.io.csv.CsvReadOptions;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.Reader;
+import java.util.TreeMap;
+
+
+/**
+ * Class that takes a String as path file
+ * to csv files
+ * and creates datasets in two formats
+ * TreeMap & Table from tablesaw.
+ */
+
+public class CsvDataLoader {
+
+    /**
+     * Simple method that parses data set from a csv file
+     * without knowing the headers of the file
+     */
+    public void parseFile(String csvFile, TreeMap dataSet) throws IOException {
+
+        Reader csvData = new FileReader(csvFile);
+
+        CSVParser parser = CSVParser.parse(csvData, CSVFormat.DEFAULT);
+
+        for (CSVRecord record : parser) {
+
+            for (String field : record) {
```

Comment:
To be honest with you I don't understand what you do here :) :). Also I thought you agreed we proceed as in the playground branch but I can't see the implementations....

Reply:
will do that. I didn't see you last comment about the agreement on playground implementation. the above implementation is not hard to understand. it's standard with commons apache csv.